### PR TITLE
Ability to add additional variables to a timeseries dsg file.

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
I needed this functionality to do some stuff for the NWC and had been planning on adding it for a while. The other alternative is to allow passing in lists of variables and metadata. I kind of like this method as it gives more control to the calling software... opinions?

Have a look at the "add var to existing file" commit for a good diff. The first commit includes a major reindent that makes the diff suck.

### CODE REVIEW

File: R/write_timeseries_dsg.R [hunk 1]
```diff
@@ -66,79 +70,102 @@ write_timeseries_dsg = function(nc_file, station_names, lats, lons, times, data,
...
+		# Open existing file.
+		nc<-nc_open(nc_file, write = TRUE)
+		data_vars = list()
+		data_vars[[1]] = ncvar_def(data_name, data_unit, dim=list(nc$dim$time, nc$dim$station), prec=data_prec, 
+															 longname=data_metadata[['long_name']], missval=-999)
+		ncvar_add(nc, data_vars[[1]])
+		nc_close(nc)
+		nc<-nc_open(nc_file, write = TRUE)
+		putDataInNC(nc,nt,n,data_name,data)
+		nc_close(nc)
```
Comment (lawinslow) [inline R/write_timeseries_dsg.R]: Would you want attributes to be handled in this case too?

File: R/write_timeseries_dsg.R [hunk 2]
```diff
@@ -66,79 +70,102 @@ write_timeseries_dsg = function(nc_file, station_names, lats, lons, times, data,
...
-	ncatt_put(nc, 'lon', 'standard_name', 'longitude')
 	
-	if(!is.na(alts[1])){
-		ncatt_put(nc, 'alt', 'standard_name', 'height')
+	if(add_to_existing) {
+		# Open existing file.
+		nc<-nc_open(nc_file, write = TRUE)
+		data_vars = list()
+		data_vars[[1]] = ncvar_def(data_name, data_unit, dim=list(nc$dim$time, nc$dim$station), prec=data_prec, 
+															 longname=data_metadata[['long_name']], missval=-999)
```
Comment (lawinslow) [inline R/write_timeseries_dsg.R]: And so this assumes all stations are already defined in the NC file? I'd just expand the documentation a bit to be clear what functionality "add" covers.

Comment (dblodgett-usgs) [inline R/write_timeseries_dsg.R]: Yeah. I added a note to the documentation.

File: R/write_timeseries_dsg.R (hunk 1 above)
Comment (dblodgett-usgs) [inline R/write_timeseries_dsg.R]: This is really just so you can tack on a bunch more variables. Don't want to get carried away right now.

**Pull request merged**